Dockerfile.builder

```diff
@@ -0,0 +1,25 @@
+FROM centos:7
+MAINTAINER "Aslak Knutsen <aslak@redhat.com>"
+ENV LANG=en_US.utf8
+
+# Some packages might seem weird but they are required by the RVM installer.
+RUN yum --enablerepo=centosplus install -y \
```

Comment:
wdyt about adding `strace` to the list of installed tools ? It could help us to debug future issues in a container.